wolfcrypt/src/asn.c

```diff
@@ -9697,7 +9703,9 @@ static int DecodeBasicOcspResponse(byte* source, word32* ioIndex,
             return ASN_PARSE_E;
 
         InitDecodedCert(&cert, resp->cert, resp->certSz, heap);
-        ret = ParseCertRelative(&cert, CERT_TYPE, VERIFY, cm);
+        /* Don't verify if we don't have access to Cert Manager. */
+        ret = ParseCertRelative(&cert, CERT_TYPE,
```

Comment:
If the function in question never needs to turn off verification like you mention above, then please NEVER NEVER turn off verification in this function.